doc/getting-started/getting-started.rst

```diff
@@ -438,7 +426,7 @@ To generate a scenario configuration file and launch script:
          :class: drop-shadow
 
    #. In the dialog box, select **shared** as the default scenario setting and
-      then click **OK**.
+      then click **OK**. This sample default scenario defines six User VMs.
```

Comment:
Recommend mentioning the 6 User VMs in the GSG intro, so that line 429 isn't the first time. Example: 

This guide will help you get started with ACRN. We'll show how to prepare a
build environment on your development computer. Then we'll walk through the
steps to set up a simple ACRN configuration on a target system. We'll use
the ACRN predefined **shared** scenario, which contains an ACRN hypervisor,
Service VM, and six User VMs, but we'll launch only one of the User VMs as
illustrated in this figure: